src/Extension/HeadingPermalink/HeadingPermalinkExtension.php

```diff
@@ -31,6 +31,8 @@ public function configureSchema(ConfigurationBuilderInterface $builder): void
             'max_heading_level' => Expect::int()->min(1)->max(6)->default(6),
             'insert' => Expect::anyOf(HeadingPermalinkProcessor::INSERT_BEFORE, HeadingPermalinkProcessor::INSERT_AFTER)->default(HeadingPermalinkProcessor::INSERT_BEFORE),
             'id_prefix' => Expect::string()->default('content'),
+            'attach_heading' => Expect::bool()->default(false),
```

Comment:
I prefer then `apply_id_to_heading`, because the possibility to allow a string (even if it is a constant) would imply that there is a choice, but so far, there is only yes or no.